language-sql.md

```diff
@@ -3,11 +3,33 @@
 ### Minimum Requirements
 
 - MySQL keywords in uppercase
-- Column names in lowercase
+    E.g.: 
```

Comment:
? How about writing “example”?

I don’t know for you, but “e.g.” or “i.e.” are always confusing to me. Although I get some sort of understanding, I find it hard to read.

Another way worth considering would be the **Valid** and **Not valid** pattern existing in [other style–guides](https://github.com/juwai/style-guide/blob/master/language-global.md). I already wrote something like that in the past, borrowing to [jscs documentation](http://jscs.info/rule/requireOperatorBeforeLineBreak), and @mark-juwai reminded me it’s a good way to be more explicit.

Maybe we should follow the same kind of structure for examples and counter-examples?